ipn/serve.go

```diff
@@ -163,6 +163,31 @@ func (sc *ServeConfig) GetTCPPortHandler(port uint16) *TCPPortHandler {
 	return sc.TCP[port]
 }
 
+// HasPathHandler returns whether if ServeConfig has at least
+// one path handler, including foreground configs.
+func (sc *ServeConfig) HasPathHandler() bool {
+	if sc.Web != nil {
+		for _, webServerConfig := range sc.Web {
+			for _, httpHandler := range webServerConfig.Handlers {
+				fmt.Println(">>>>", httpHandler.Path != "")
```

Comment:
use "XXX" for temporary debug lines to make sure they never get submitted, or to signal to reviewers that they're temporary and will be deleted